src/ManagedServices/ManagedServices/help/Az.ManagedServices.md

```diff
@@ -0,0 +1,31 @@
+---
+Module Name: Az.ManagedServices
+Module Guid: {{ Update Module Guid }}
+Download Help Link: {{ Update Download Link }}
+Help Version: {{ Update Help Version }}
+Locale: {{ Update Locale }}
+---
+
+# Az.ManagedServices Module
+## Description
+This feature is used by customers of Managed Service Providers (MSPs). Customers give an MSP the ability to manage their subscription. In addition to granting access, the customer can also remove access or view existing access delegations. MSPs are able able to view the list of customers who have granted them access to subscriptions. There are two objects involved in this process: A registration definition which identifies the MSP and the role definitions granted to the MSP. The MSP can optionally define this object using a Managed Services marketplace offering Access assignments which associate a subscription with the definition.
+
+## Az.ManagedServices Cmdlets
+### [Get-AzManagedServicesAssignment](Get-AzManagedServicesAssignment.md)
+Gets a list of the registration assignments.
+
+### [Get-AzManagedServicesDefinition](Get-AzManagedServicesDefinition.md)
+Gets a list of the registration definitions.
+
+### [New-AzManagedServicesAssignment](New-AzManagedServicesAssignment.md)
+Creates or updates a registration assignment.
```

Comment:
Can it actually update?  If so, the verb should be set

Reply:
No it doesnt. updated the text accordingly.